extension/scripts/background.js

```diff
@@ -12,6 +12,45 @@ if (localStorage.getItem('enabled') === null) {
 	enabled = localStorage.getItem('enabled') === 'true';
 }
 
+let blackListPromise;
+(function updateBlacklist () {
+	blackListPromise = fetch(
+		'http://bertha.ig.ft.com/view/publish/gss/1_f-cR8VbcujCsjxTLyKVRcG9yp_qUbT0-FrdmwBp0OM/blacklist'
+	).then(response => response.json());
+
+	// Once an hour check for new black listed sites.
+	setTimeout(updateBlacklist , 3600 * 1000);
+}());
+
+function checkIsNotBlackListed (url) {
```

Comment:
This function is asking a question ("Are you not blacklisted?"), I think it should return a boolean, true if you are not blacklisted, false if you are. Currently this function is throwing if the url is blacklisted, which causes the caller of this function to have to catch this exception. I don't think throwing is the correct approach here as we expect some urls to be in the blacklist and some others to not be in the blacklist.